app/grandchallenge/reader_studies/models.py

```diff
@@ -748,6 +748,14 @@ def list_url(self):
     def civ_sets_related_manager(self):
         return self.display_sets
 
+    @property
+    def interfaces(self):
+        return ComponentInterface.objects.filter(
+            slug__in=self.display_sets.exclude(
+                values__isnull=True
+            ).values_list("values__interface__slug", flat=True)
+        ).distinct()
```

Comment:
Same comment as for the archive property

Reply:
Done